packages/@react-types/table/src/index.d.ts

```diff
@@ -129,9 +129,16 @@ export interface CellProps {
 export type CellElement = ReactElement<CellProps>;
 export type CellRenderer = (columnKey: Key) => CellElement;
 
+// TODO: perhaps defined `rows`, etc here so that we can table specific definitions
 export interface TableCollection<T> extends GridCollection<T> {
+  // TODO perhaps elaborate on this? maybe not clear enought, essentially returns the table header rows (e.g. in a tiered headers table, will return the nodes containing the top tier column, next tier, etc)
+  /** A list of header row nodes in the table. */
   headerRows: GridNode<T>[],
+  /** A list of column nodes in the table. */
   columns: GridNode<T>[],
+  // TODO perhaps elaborate here, link to rowheader docs? https://www.digitala11y.com/rowheader-role/?
```

Comment:
I don't think that's actual docs? though mozilla has a broken link for the role https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Row_Role
but we should really be looking for these links i think https://www.w3.org/TR/wai-aria-1.1/#rowheader